app/experimenter/nimbus-ui/src/components/PageRequestReview/index.stories.tsx

```diff
@@ -3,134 +3,106 @@
-      publishStatus: NimbusExperimentPublishStatus.WAITING,
-    });
+import { mock, Subject } from "./mocks";
+
+const storyWithExperimentProps = (
+  props: Partial<getExperiment_experimentBySlug | null>,
+  storyName?: string,
+) => {
+  const story = () => {
+    const { mock } = mockExperimentQuery("demo-slug", props);
     return <Subject {...{ mocks: [mock] }} />;
-  })
-  .add("error", () => (
-    <RouterSlugProvider mocks={[mock]}>
-      <PageRequestReview polling={false} />
-    </RouterSlugProvider>
-  ));
-
-storiesOf(
-  "pages/RequestReview/EXP-1143; end experiment review refactor",
-  module,
-)
-  .addDecorator(withLinks)
-  .add("review not requested", () => <SubjectEXP1143 />)
-  .add("review requested, user can review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        canReview: true,
-      }}
-    />
-  ))
-  .add("review pending in Remote Rettings, user can review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        approvalEvent: mockChangelog("def@mozilla.com"),
-        canReview: true,
-      }}
-    />
-  ))
-  .add("review timed out in Remote Settings, user can review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        approvalEvent: mockChangelog("def@mozilla.com"),
-        timeoutEvent: mockChangelog("ghi@mozilla.com"),
-        canReview: true,
-      }}
-    />
-  ))
-  .add("review requested, user cannot review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        canReview: false,
-      }}
-    />
-  ))
-  .add("review pending in Remote Settings, user cannot review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        approvalEvent: mockChangelog("def@mozilla.com"),
-        canReview: false,
-      }}
-    />
-  ))
-  .add("review timed out in Remote Settings, user cannot review", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        approvalEvent: mockChangelog("def@mozilla.com"),
-        timeoutEvent: mockChangelog("ghi@mozilla.com"),
-        canReview: false,
-      }}
-    />
-  ))
-  .add("review rejected", () => (
-    <SubjectEXP1143
-      {...{
-        reviewRequestEvent: mockChangelog(),
-        approvalEvent: mockChangelog("def@mozilla.com"),
-        rejectionEvent: mockChangelog(
-          "ghi@mozilla.com",
-          "It's bad. Just start over.",
-        ),
-        canReview: true,
-      }}
-    />
-  ));
+  };
+  story.storyName = storyName;
+  return story;
+};
+
+export default {
+  title: "pages/RequestReview",
+  component: Subject,
+  decorators: [withLinks],
+};
+
+export const draftStatus = storyWithExperimentProps({
+  status: NimbusExperimentStatus.DRAFT,
+});
+
+export const previewStatus = storyWithExperimentProps({
+  status: NimbusExperimentStatus.PREVIEW,
+});
```

Comment:
Tackling a bit of issue #5004 here by refactoring all the stories to v6 format, which also helps DRY some things up.